Header header
float32[] line_left
float32[] line_right
float32[] x_left
float32[] y_left
float32[] x_right
float32[] y_right
